Digipost/APIClient.swift

```diff
@@ -99,17 +99,18 @@ class APIClient : NSObject, NSURLSessionTaskDelegate, NSURLSessionDelegate, NSUR
         return urlString
     }
 
-    func validateTokensThenPerformTask(task: NSURLSessionTask) {
-        validateOAuthToken(kOauth2ScopeFull) {
-            self.updateAuthorizationHeader(kOauth2ScopeFull)
+    func validate(#token: OAuthToken?, thenPerformTask task: NSURLSessionTask) {
```

Comment:
I think i can remove the method, its basically deprecated and replaced with the more clear validate:then { }